cypress/integration/starterTest.spec.js

```diff
@@ -0,0 +1,8 @@
+describe('My First Test', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+  it('Does not do much!', () => {
+    expect(true).to.equal(false)
+  })
+  })
```

Comment:
This test is a good start, and will inevitably fail.  Is this meant as a test of functionality of cypress?  If so, it's probably fine to delete it after verifying that it's working.